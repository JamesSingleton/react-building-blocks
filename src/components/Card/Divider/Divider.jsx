import React from 'react';
import styled from 'styled-components';

const StyledDivider = styled.div`
  width: 100%;
  border-bottom: 1px solid #dfe5eb;
`;

const Divider = () => <StyledDivider />;

export default Divider;
