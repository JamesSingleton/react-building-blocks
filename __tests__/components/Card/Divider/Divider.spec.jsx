import React from 'react';
import { mount } from 'enzyme';
import Divider from '../../../../src/components/Card/Divider';

describe('Divider', () => {
  it('should render', () => {
    const renderedDivider = mount(<Divider />);
    expect(renderedDivider).toMatchSnapshot();
  });
});
