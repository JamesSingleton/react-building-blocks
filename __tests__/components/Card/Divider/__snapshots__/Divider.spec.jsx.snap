// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider should render 1`] = `
<Divider>
  <styled.div>
    <StyledComponent
      forwardedComponent={
        Object {
          "$$typeof": Symbol(react.forward_ref),
          "attrs": Array [],
          "componentStyle": ComponentStyle {
            "componentId": "sc-bdVaJa",
            "isStatic": true,
            "lastClassName": "ljuHUZ",
            "rules": Array [
              "
  width: 100%;
  border-bottom: 1px solid #dfe5eb;
",
            ],
          },
          "displayName": "styled.div",
          "foldedComponentIds": Array [],
          "render": [Function],
          "styledComponentId": "sc-bdVaJa",
          "target": "div",
          "toString": [Function],
          "warnTooManyClasses": [Function],
          "withComponent": [Function],
        }
      }
      forwardedRef={null}
    >
      <div
        className="sc-bdVaJa ljuHUZ"
      />
    </StyledComponent>
  </styled.div>
</Divider>
`;
